Marlin/Marlin_main.cpp

```diff
@@ -2419,8 +2425,13 @@ static void clean_up_after_endstop_or_probe_move() {
 
     #elif ENABLED(AUTO_BED_LEVELING_UBL)
 
+      #if ENABLED(UBL_DELTA)
+        if (( ubl.state.active ) && ( ! enable )) {   // leveling from on to off
+          planner.unapply_leveling(current_position);
```

Comment:
@oldmcg @Roxy-3D
One bug: This is exactly backwards. We want to "apply leveling" to the current position before disabling leveling. This causes the current position to contain the correct physical position.